lib/modules/datasource/docker/index.ts

```diff
@@ -464,6 +464,65 @@ export class DockerDatasource extends Datasource {
     }
   }
 
+  @cache({
+    namespace: 'datasource-docker-imageconfig',
+    key: (
+      registryHost: string,
+      dockerRepository: string,
+      configDigest: string
+    ) => `${registryHost}:${dockerRepository}@${configDigest}`,
+    ttlMinutes: 1440 * 28,
+  })
+  public async getImageConfig(
+    registryHost: string,
+    dockerRepository: string,
+    configDigest: string
+  ): Promise<HttpResponse | null | undefined> {
+    logger.debug(
+      `getImageConfig(${registryHost}, ${dockerRepository}, ${configDigest})`
+    );
+    try {
+      const headers = await getAuthHeaders(
+        this.http,
+        registryHost,
+        dockerRepository
+      );
+      // istanbul ignore if: Should never be happen
+      if (!headers) {
+        logger.debug('No docker auth found - returning');
+        return undefined;
+      }
+      const url = `${registryHost}/v2/${dockerRepository}/blobs/${configDigest}`;
+      return await this.http.get(url, {
```

Comment:
```suggestion
      return await this.http.getJson<ImageConfig>(url, {
```

use `getJson` and let `got` do the json parsing. needs a proper type declaration (can be minimal)